public/app/ui/pages/search/SearchPage.js

```diff
@@ -109,6 +109,11 @@
                     );
                 };
 
+                this.ready = function (callback) {
```

Comment:
This is still necessary for the case where the app is initially opened at a search page initial URL (i.e. not a results URL, but just the URL of the search form itself with no initial query).